Examples/TicTacToe/App/Info.plist

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This got lost in the other changes. For some reason it still built fine if it already existed in a sim, but in fresh sims installation failed.